oxidation/libparsec/crates/core_fs/src/storage/chunk_storage.rs

```diff
@@ -485,8 +489,8 @@ mod tests {
         );
 
         let local_chunk_ids = chunk_storage.get_local_chunk_ids(&chunk_ids).await.unwrap();
-        let set0: HashSet<_, RandomState> = HashSet::from_iter(local_chunk_ids.iter());
-        let set1: HashSet<_, RandomState> = HashSet::from_iter(chunk_ids.iter());
+        let set0 = HashSet::<&ChunkID>::from_iter(local_chunk_ids.iter());
+        let set1 = HashSet::<&ChunkID>::from_iter(chunk_ids.iter());
```

Comment:
isn't `type` infered by `local_chunk_ids` ?

Reply:
Apparently no, I was getting error that `set0` & `set1` need to be typed

Reply:
Maybe you could consider
```rust
let set0 = local_chunk_ids.iter().collect::<HashSet<_>>();
let set1 = chunk_ids.iter().collect::<HashSet<_>>();
```
And I tried, only 1 of both is required, but it's strange :eyes:

Reply:
You prefer the collect form ?

Reply:
`collect` avoid the use of `FromIter` `trait` and `2` characters less than `from_iter`

Reply:
done

Reply:
There's a problem with `trait FromIterator` ?

Reply:
Nope